### PR TITLE
feat(web): refactor page header

### PR DESCRIPTION
**Pull request by issmail-basel**
# Overview
Add needed params to `PageHeader` atom.
Remove any duplicated header code.

### CODE REVIEW

**Review by nourbalaha — APPROVED** [review]
Approved!

File: web/src/components/atoms/PageHeader/index.tsx [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+import { PageHeader, PageHeaderProps } from "antd";
 
-export default PageHeader;
+export type Props = PageHeaderProps & {
+  backgroundColor?: string;
+};
+
+const Header: React.FC<Props> = ({ ...props }) => {
+  return <StyledPageHeader {...props} backgroundColor={props.backgroundColor} />;
+};
```
Comment (KaWaite) [inline web/src/components/atoms/PageHeader/index.tsx:10]:
```suggestion
const Header: React.FC<Props> = ({ backgroundColor, ...props }) => {
  return <StyledPageHeader {...props} backgroundColor={backgroundColor} />;
};
```

File: web/src/components/atoms/PageHeader/index.tsx [hunk 2]
```diff
@@ -1,3 +1,14 @@
-import { PageHeader } from "antd";
+import styled from "@emotion/styled";
+import { PageHeader, PageHeaderProps } from "antd";
 
-export default PageHeader;
+export type Props = PageHeaderProps;
+
+const Header: React.FC<Props> = ({ ...props }) => {
+  return <StyledPageHeader {...props} />;
+};
```
Comment (KaWaite) [inline web/src/components/atoms/PageHeader/index.tsx:8]:
```suggestion
const Header: React.FC<Props> = props => {
  return <StyledPageHeader {...props} />;
};
```

Comment (issmail-basel) [reply web/src/components/atoms/PageHeader/index.tsx]: https://github.com/reearth/reearth-cms/blob/375929716ec3153a271f4be664babb9212bf5954/web/src/components/atoms/PageHeader/index.tsx#L1-L14

**Review by KaWaite — APPROVED** [review]

**Pull request merged**